frontend/aml_dashboard/src/index.js

```diff
@@ -258,6 +261,7 @@ collaborative_status.title = 'AML Collaborative Learning Status';
 const search_statistics = button('Search for statistics');
 search_statistics.title = 'AML Statistics Fetcher'
 
+const control_value = text('No statistics received');
```

Comment:
Because `collaborative_status` is used for all the sections in the code, therefore it is not a reliable variable to handle the success and failure of receiving statistics. on the other hand, `statistics_received` includes  the variable `name` which seems to be a local variable, therefore I don't think i can include it in `request_model`.